EastAgile/twitterbot
# [f] 847 - posts by twitterbot should always include a in_reply_to_status_id parameter with the originating tweet ID

Description
--
* Add `in_reply_to` param to tweet API method

How to test
--
* Use `client.tweet` with `in_reply_to` param set to a tweet ID AND mention the tweets' owner's screen name in the reply text should result in a `conversation` on the replied tweet user timeline (a `View conversation` link is visible)

[#120698847] https://www.pivotaltracker.com/story/show/120698847
